.gitignore

```diff
@@ -8,3 +8,4 @@ resources/
 credentials.json
 .cogs
 .git-credentials
+.git-credentials
```

Comment:
This is duplicate -- DROID probably added it by mistake.

Reply:
Fixed